api/test/nostd/BUILD

```diff
@@ -16,6 +16,7 @@ cc_test(
     ],
     deps = [
         "//api",
+        "@com_github_google_benchmark//:benchmark",
```

Comment:
For here and other benchmarks, can we keep the benchmarks in separate CC files, for the bazel build?

Reply:
Yes, I will do it.

Reply:
Marking as Resolved. Moving Benchmarks into separate PR.